lib/zclient.c

```diff
@@ -2838,6 +2895,15 @@ static int zclient_read(struct thread *thread)
 			(*zclient->vxlan_sg_del)(command, zclient, length,
 						    vrf_id);
 		break;
+	case ZEBRA_MLAG_PROCESS_UP:
+		zclient_mlag_process_up(command, zclient, length, vrf_id);
```

Comment:
Most registered functions so far were taking all these 4 arguments. It will be good to pass all these arguments to registered function on zclient->mlag_process_up, zclient->mlag_process_up. Is there any reason we dont pass all of them? good chance that some client which will register for these events may need more than what you pass to them(like vrf). IMO we can make the prototype standard to pass all of the arguments to registering function.

Reply:
MLAG Prcess UP & DOWN is. global Notification, not bounded to any VRF. Because of that nt passing any info to clients. Based on the Notifications clients needs to do some cleanup.

Reply:
The comment was general for all the registered functions.